### DISCUSSION TITLE
ModuleNotFoundError: No module named 'encodec'

### DISCUSSION THREAD

**Discussion by Deadstarrr** [suno-ai/bark#240] [General]
I installed bark through the C0untFloyd GUI and I can start Bark by clicking on windows_start.bat, but that sometimes reinstalls everything and takes forever. If I try to launch it with StartBark.bat instead, then I get an error saying ModuleNotFoundError: No module named 'encodec'.

Anyone else get this error and know how to fix it?

**Comment by gkucsko** [comment]
if encodec isn't installed then i'd try `pip install encodec`. but not super sure about the reason, haven't looked at the gui code sorry

**Comment by C0untFloyd** [comment]
Please just be patient and let the installation run through just once. It might take some time to download all packages & models but once this is finished, it won't happen again.
If you still have problems, consider opening an issue in the correct [fork](https://github.com/C0untFloyd/bark-gui/issues) don't bother the good Suno people 😉
